### PR TITLE
Add sipecam dataset information

### PR DESCRIPTION
**Pull request by caroacostatovany**
*Issue #, if available:*

*Description of changes:*
Add sipecam dataset to the aws open data repository.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/sipecam.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Name: SiPeCaM (Sitios Permanentes de la Calibración y Monitoreo de la Biodiversidad)
+Description: The SiPeCaM goal is to create a data source that allows to evaluate changes in the biodiversity state, considering key aspect of how does the ecosystem behaves.
+Documentation: https://github.com/CONABIO/sipecam-open-data/blob/main/docs/SiPeCaM.md
+Contact: "Michael F. Schmidt [mschmidt@conabio.gob.mx](mschmidt@conabio.gob.mx)"
+ManagedBy: "[Conabio](https://monitoreo.conabio.gob.mx/)"
+UpdateFrequency: SiPeCaM offers data releases once the biologist recollect the data from the protected natural areas (every 3 months)
+Tags:
```
Comment (jflasher) [inline datasets/sipecam.yaml:7]: Please stick to tags already in https://github.com/awslabs/open-data-registry/blob/main/tags.yaml. If you think that new tags should be added, please make the changes in tags.yaml and include them in this pull request.

File: datasets/sipecam.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - biodiversity
+  - fauna
+  - animals audio
+  - protected areas
+License: Creative Commons with attribution
```
Comment (jflasher) [inline datasets/sipecam.yaml:12]: Could you link to an actual license. I think in this case you are looking for https://creativecommons.org/licenses/by/4.0/ ?

File: datasets/sipecam.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Resources:
+  - Description: Audio, video and image representation of mexican fauna
+    ARN: arn:aws:s3:::sipecam-open-data
```
Comment (jflasher) [inline datasets/sipecam.yaml:15]: This bucket does not look to be publicly accessible?

File: datasets/sipecam.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    Region: us-east-1
+    Type: s3 Bucket
+  Publications:
```
Comment (jflasher) [inline datasets/sipecam.yaml:18]: Looks like you lost the `DataAtWork` field. Please see a dataset like https://github.com/awslabs/open-data-registry/blob/main/datasets/1000-genomes-data-lakehouse-ready.yaml to make sure you get the right formatting for the listing.

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @caroacostatovany thanks for the PR! I made a few change requests and had a question or two.

File: datasets/sipecam.yaml (hunk 3 above)
Comment (caroacostatovany) [reply datasets/sipecam.yaml:15]: Can this be private with access request? or it should be completely publicly accessible?

**Review by cstner — COMMENTED** [review]
You will need to change the Region on line 18 from us-east-1 to us-west-2, as the build is failing checking the bucket location: "The region for bucket sipecam-open-data is listed as us-east-1 but is actually us-west-2"

**Comment by caroacostatovany** [comment]
> 

Thank you for letting me know. The change has been done

**Review by cstner — APPROVED** [review]

**Pull request merged**